client/src/components/Profile/index.tsx

```diff
@@ -72,13 +76,82 @@ export default class Profile extends React.Component<IProfileProps> {
         this.setState({modal: !this.state.modal});
     }
 
+    public followUrlUser = (): void => {
+        const body = {_id: this.props.user._id};
+        this.props.followUser(body);
+    }
+
+    public unfollowUrlUser = (): void => {
+        const body = {_id: this.props.user._id};
+        this.props.unfollowUser(body);
+    }
+
+    public dynamicButton = (): any => {
+        const loggedUserAlreadyFollowUrlUser =
+            this.props.user.followers.filter((follower: any) => follower === this.props.loggedId);
+        const urlUserAlreadyFollowLoggedUser =
+            this.props.user.following.filter((follower: any) => follower === this.props.loggedId);
```

Comment:
move to function filter:
```
const filterDataList = (data) => data.filter((follower: any) => follower === this.props.loggedId);
```

and use 

```
const loggedUserAlreadyFollowUrlUser = filterDataList(this.props.user.followers);
```
etc

Reply:
fixed